wiki/People/peppy/en.md

```diff
@@ -9,4 +9,9 @@ tags:
 
 **peppy** (stylised with a lowercase "p") is the online alias of Dean Herbert who is the creator of the osu! project and its primary [developer](/wiki/People/The_Team/Developers).
 
-<!-- TODO: Add links and stuff -->
+## Links
+
+- **[Web site](https://ppy.sh/)**
```

Comment:
```suggestion
- **[Website](https://ppy.sh/)**
```
maybe add his GitHub profile as well? 🤔

Reply:
ehm.. idk

Reply:
i don't think his GH profile is necessary.

Reply:
I'd still record that as important -- the organization hosts a lot of things

Reply:
@TicClick  what else can i add?

Reply:
just that -- I'll add it myself now